systemjs-bundle/task.js

```diff
@@ -14,10 +13,10 @@ const packageDependencies = deps.map(matchStartsWith);
 const packageName = packageInfo.name;
 const SRC_EXT_GLOB = ".{jsx,ts,tsx,js}";
 
-module.exports = (gulp, { webpackConfig, distName, modules = [] }) => {
+module.exports = (gulp, { webpackConfig, distName, modules = [] }, webpackStream, webpack) => {
```

Comment:
Let's pass the stream and the webpack as parameters in the "options" object

`
module.exports = (gulp, { webpackConfig, distName, modules = [], webpackStream, webpack }) => {
`

Reply:
We just need to verify that the version of the webpackStream used in the common tasks accepts second parameter.